README.md

```diff
@@ -430,6 +431,16 @@ succeed. If your users are coming in over the Internet (and not over LAN), this
 The public IP address is discovered via [STUN]. STUN servers can be specified with the ``JVB_STUN_SERVERS``
 option.
 
+## Build Instructions
+
+Building your own images allows you to edit the configuration files of each image individually, providing more customization for your deployment.
```

Comment:
Can you remove the extra space before the B?

Reply:
I think the space before `Build` is required for the md formatting, and I don't think there is a space before `Building`. The formatting in the diff may make it look like there is a space ;)

Reply:
Unless you are talking about the line 435? But that would conflict with the other titles in this doc.